docs/source/conf.py

```diff
@@ -59,5 +59,4 @@
 html_static_path = ["_static"]
 html_css_files = ["openkat.css"]
 
-mermaid_version = ""  # Do not fetch from the CDN
-html_js_files = ["mermaid-9.4.3.min.js"]
```

Comment:
self hosting the js files is preferable, also our csp headers do not allow for the cdn.

Reply:
Good point. I was trying this out since for some reason I got errors when using the newer version hosted locally, but forgot to remove it. I'll revert this and try something else